### PR TITLE
[RNChat] Upgrades, updates, fixes, improvements

### PR DESCRIPTION
**Pull request by ccvlad**

### CODE REVIEW

File: RNChat/src/screens/auth/AuthForm.js [hunk 1]
```diff
@@ -1,89 +1,90 @@
...
+import { showAlert } from '../../helpers/alert';
+import { AuthService } from '../../services';
+import Indicator from '../components/indicator';
 
-export default function AuthForm ({isLogin}) {
-  const [name, setName] = useState('');
-  const [password, setPassword] = useState('');
-  const [isLoader, setIsLoader] = useState(false);
+export default function AuthForm({ isLogin }) {
+	const [name, setName] = useState('');
```
Comment (DDooni) [inline RNChat/src/screens/auth/AuthForm.js:8]: tabs??

**Review by DDooni — APPROVED** [review]
Good!

File: RNChat/README.md [hunk 2]
```diff
@@ -1,44 +1,140 @@
...
+- Search application users to create 1-1 or group chat;
+- Create, update, delete group chat;
+- Send and receive messages/images;
+- Send and receive sent/delivered/read message statuses;
+- Receive push notifications.
 
 ## Documentation
 
-ConnectyCube React Native getting started - [https://developers.connectycube.com/reactnative](https://developers.connectycube.com/reactnative)
+ConnectyCube React Native SDK getting started - [https://developers.connectycube.com/reactnative](https://developers.connectycube.com/reactnativee)
```
Comment (DaveLomber) [inline RNChat/README.md:19]: typo
reactnativee -> reactnative

File: RNChat/README.md [hunk 3]
```diff
@@ -1,44 +1,140 @@
...
+{
+  "connectyCubeConfig": [
+    {
+      "appId": 0,
+      "authKey": "***",
+      "authSecret": "***"
+    },
+    {
+      ...
+      "chat": {
```
Comment (DaveLomber) [inline RNChat/README.md:104]: let's make it simper for users. Let's remove all these custom settings and keep credentials only

File: RNChat/README.md [hunk 4]
```diff
@@ -1,44 +1,140 @@
...
+      },
+      "debug": {
+        "mode": 1 // mode 0 is logs off, mode 1 -> console.log()
+      }
+    }
+  ]
+}
+```
+
+3. (Optional) If you are at [Enterprise](https://connectycube.com/pricing/) plan - provide your API server and Chat server endpoints at `connectycube_config.json` file to point the sample against your own server:
```
Comment (DaveLomber) [inline RNChat/README.md:117]: this section can be removed

File: RNChat/README.md [hunk 5]
```diff
@@ -1,44 +1,140 @@
...
+  ]
+}
+```
+
+4. Run on Android/iOS device
 
 ## Can't build yourself?
 
-Got troubles with building React Native code sample? Just create an issue at [Issues page](https://github.com/ConnectyCube/connectycube-reactnative-samples/issues)
+Got troubles with building React Native code sample? Just create an issue at [Issues page](https://github.com/ConnectyCube/connectycube-reactnative-samples/issues) - we will create the sample for you. For FREE!
```
Comment (DaveLomber) [inline RNChat/README.md:140]: no need " we will create the sample for you. For FREE!"

**Review by DaveLomber — APPROVED** [review]
approved with some comments

**Pull request merged**